test/unit/lib/metrics.js

```diff
@@ -195,7 +197,9 @@ describe('lib/metrics.js', () => {
             it('increments the Prometheus counter lev_api_req_client', () => register.getSingleMetricAsString('lev_api_req_myclient').should.match(/[^0-9]1$/));
             it('increments the Prometheus counter lev_api_req_group', () => register.getSingleMetricAsString('lev_api_req_group').should.match(/[^0-9]1$/));
             it('increments the Prometheus counter lev_api_req_lookup', () => register.getSingleMetricAsString('lev_api_req_lookup').should.match(/[^0-9]1$/));
-            it('increments the Prometheus counter lev_api_req_birth_lookup', () => register.getSingleMetricAsString('lev_api_req_birth_lookup').should.match(/[^0-9]1$/));
+            it('increments the Prometheus counter lev_api_req_birth_lookup', () => register.getSingleMetricAsString('lev_api_req_birth_lookup').should.match(/\D1$/));
+            it('increments the Prometheus counter lev_api_req_birth_blocked', () => register.getSingleMetricAsString('lev_api_req_birth_blocked').should.match(/ 1$/));
+
```

Comment:
@daniel-ac-martin these changes were made to try and provoke a response with your preference! I'm surprised you haven't commented already.

Reply:
I'm afraid I can't remember enough about how these metrics work.

The space is nicest if we can actually rely on that behaviour. Regardless, we should change the other lines to match.

`\D` feels cryptic to me but maybe that's just because my regex-fu isn't up to snuff.

Reply:
`\d` === `[0-9]`
`\D` === `[^0-9]`
You could use `\s` (for whitespace) or `\b` (for word boundary) if some fuzziness were desirable.
It's a shame there doesn't seem to be an `ends With` assertion 😔

Reply:
@aarongill1 I'll leave you to decide the implementation you think is most intuitive